alvr/launcher/src/commands.rs

```diff
@@ -214,5 +211,8 @@ pub fn invoke_installer() {
     spawn_no_window(Command::new(commands::installer_path()).arg("-q"));
 
     // delete crash_log.txt (take advantage of the occasion to do some routine cleaning)
-    fs::remove_file(current_alvr_dir().unwrap().join(CRASH_LOG_FNAME)).ok();
+    fs::remove_file(alvr_filesystem_layout::crash_log(
+        &current_alvr_dir().expect("get alvr dir"),
```

Comment:
Use [`trace_err` or `trace_none`](https://github.com/alvr-org/ALVR/wiki/How-ALVR-works#logging-and-error-management) please. No need to panic if we are returning a `StrResult`.

Reply:
It seems the web diff just messed up a bit and made me think this function was returning `StrResult` when, in fact, it isn't.